src/components/leaderboard.js

```diff
@@ -20,6 +22,7 @@ AFRAME.registerComponent('leaderboard', {
 
     challengeId: {default: ''},
     difficulty: {default: ''},
+    beatmapCharacteristic: { default: '' },
```

Comment:
what does beatmapCharacteristic mean?

Reply:
`characteristic` is basically a part of the difficulty, so for example there could be the difficulties `Expert` and `ExpertPlus` with a `Standard` characteristic, but also a `ExpertPlus` difficulty with a `OneSaber` characteristic in the same beatmap. basically different variants

Reply:
ah maybe `playMode` is a better name? `characteristic` maybe a bit generic?

Reply:
I decided to use `characteristic` because that's the name used by the beatsaver API and official beatmap format

Reply:
If not a lot of trouble I think it's ok to rename so we don't inherit bad naming :)

Reply:
I would prefer to keep it this way.